ledger-tool/src/blockstore.rs

```diff
@@ -113,28 +109,28 @@ fn analyze_column<
     Ok(())
 }
 
-fn analyze_storage(database: &Database) -> Result<()> {
+fn analyze_storage(blockstore: &Blockstore) -> Result<()> {
```

Comment:
I considered ripping this leder-tool function out, but deemed it out of scope for this PR. So, I did the bare minimal to keep it functional. The function I added, `Blockstore::iterator_cf()`, to support is behind dcou so I feel there is little harm with it + I'll be tweaking some of the iterator stuff soon for https://github.com/anza-xyz/agave/issues/3850

Reply:
Also, the "prettier" names was extra bloat so I just decided to use the column names as we store them in Rocks to remove extra hard-coding